pimd/pim6_cmd.c

```diff
@@ -215,21 +215,55 @@ DEFPY (no_ipv6_pim_register_suppress,
 
 DEFPY (interface_ipv6_pim,
        interface_ipv6_pim_cmd,
-       "ipv6 pim",
+       "ipv6 pim [passive$passive]",
        IPV6_STR
-       PIM_STR)
+       PIM_STR
+       "Disable exchange of protocol packets\n")
 {
-	return pim_process_ip_pim_cmd(vty);
+	int ret;
+	VTY_DECLVAR_CONTEXT(interface, ifp);
+	struct pim_interface *pim_ifp;
+
+	ret = pim_process_ip_pim_cmd(vty);
+
+	if (ret != NB_OK)
+		return ret;
+
+	pim_ifp = ifp->info;
+	if (!pim_ifp)
+		return CMD_WARNING_CONFIG_FAILED;
+
+	if (passive)
+		pim_ifp->pim_passive_enable = true;
+
```

Comment:
@patrasar , sorry, I was concise in my previous comment. It is probably better to be explicit in all of the config cases of passive  and have

```
if (passive)
		pim_ifp->pim_passive_enable = true;
else
		pim_ifp->pim_passive_enable = false;
```

Reply:
or better 
```
pim_ifp->pim_passive_enable = passive;
```